### ISSUE TITLE
Formalize testing for TAP

### ISSUE THREAD

**Issue by vrurg** [SqrtNegInf/Rosettacode-Raku-Smoke#4]
Would it be possible to consider turning the project into a module with formal testing process resulting in valid TAP? The purpose is to register the module in Raku ecosystem and help in release testing of Rakudo.

My point is that every Rakudo release is preceded with a run of Blin which tests all ecosystem and CPAN modules. Having Rosettacode included into the process would be extremely helpful.

**Comment by SqrtNegInf** [comment]
Yes this is a good idea, and should be do-able, with some limitations.
Will start thinking on this...

What's a good module name for a 'collection of disparate tasks'?  With an
eye towards allowing for expansion:

Collection::Rosettacode
Collection::Euler  (for projecteuler.net tasks)

But seeing as I've never even released a module before, I'm open to
suggestions.

**Comment by vrurg** [comment]
I would preserve `Collection` for modules which would do some kind of data processing. We currently have two prefixes with fuzzy meaning: `Foo::` and `Acme::`. The former is more for testing different kind of fail conditions. But the latter could be the one. Something like `Acme::Collection::Rosettacode` sounds about right for me.
